src/Microsoft.AspNetCore.Hosting/Startup/ExceptionPage/Views/ErrorPage.js

```diff
@@ -0,0 +1,192 @@
+﻿(function (window, undefined) {
```

Comment:
Unless it's a crazy amount of work to make it vary by repo... I'm not *too* worried if there's some extra stuff in here...